php_phongo.c

```diff
@@ -234,8 +234,27 @@ static void php_phongo_log(mongoc_log_level_t log_level, const char *log_domain,
 
 /* }}} */
 
+/* Splits a namespace name into the database and collection names, allocated with estrdup. */
+static bool phongo_split_namespace(const char *namespace, char **dbname, char **cname) /* {{{ */
```

Comment:
This was moved up since `phongo_cursor_init_for_query()` depends on it. We don't have any other early declarations in `php_phongo.c`, and I opted not to start just for this.